android/source/VideoLocker/src/org/edx/mobile/view/custom/ETextView.java

```diff
@@ -36,9 +36,10 @@ private void processAttrs(Context context, AttributeSet attrs) {
         try {
             // check for the font attribute and setup font
             String fontFileName = a.getString(R.styleable.custom_view_font);
-            /*if(fontFileName==null){
-                fontFileName = attrs.getAttributeValue(null, "font");
-            }*/
+            if(fontFileName==null){
+                //If font is not set for ETextView, set default font
+                fontFileName = "OpenSans-Regular.ttf";
+            }
```

Comment:
@shahidtamboli I assume this do not make any unwanted changes to the font of any text in the app, as this applies a default font to the entire app.